src/git-operations.ts

```diff
@@ -9,6 +9,47 @@ const HEAD = 'HEAD';
 type DiffMap = Map<string, string[]>;
 const DIFFS: DiffMap = new Map();
 
+/**
+ * Gets the HTTPS URL of the current GitHub remote repository. Assumes that
+ * the git config remote.origin.url string matches one of:
+ *
+ * - https://github.com/OrganizationName/RepositoryName
+ * - git@github.com:OrganizationName/RepositoryName.git
```

Comment:
Any idea when each of these version are used? The `@action/checkout` docs indicate that `github.repository` is the default, but I haven't tracked down what that is yet.

Reply:
Hm, I think it's simply dictated by whether `git` is configured to use SSH or HTTP/HTTPS. If you search for `github.repository` [here](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context), you'll see that it's just `OrganizationName/RepositoryName`.

[`actions/checkout`](https://github.com/actions/checkout/) supports either HTTPS or SSH, and the one it uses is determined by the inputs you provide to it, so for present purposes I think we're good so long as we support both?

Reply:
>actions/checkout supports either HTTPS or SSH, and the one it uses is determined by the inputs you provide to it

I couldn't find confirmation of this in their docs, but I found some code that indicates that this is true! https://github.com/actions/checkout/blob/main/src/url-helper.ts